code/modules/projectiles/projectile.dm

```diff
@@ -547,6 +547,8 @@ var/list/impact_master = list()
 
 /obj/item/projectile/proc/reset()
 	starting = get_turf(src)
+	if(isnull(starting))
```

Comment:
How is it spawning in nullspace?

Reply:
not spawning there
https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/game/turfs/turf.dm#L269